src/chrome/internal/locations/location.ts

```diff
@@ -0,0 +1,157 @@
+import * as Validation from '../../../validation';
+import { IScript } from '../scripts/script';
+import { ISource } from '../sources/source';
+import { ILoadedSource } from '../sources/loadedSource';
+import { logger } from 'vscode-debugadapter';
+import { ColumnNumber, LineNumber, URLRegexp } from './subtypes';
+import { CDTPScriptUrl } from '../sources/resourceIdentifierSubtypes';
+import { IResourceIdentifier, parseResourceIdentifier, URL } from '../sources/resourceIdentifier';
+
+export type integer = number;
+
+export class Coordinates {
```

Comment:
Could we match vscode on this and call it a `Position`?

Reply:
Sure. I'll rename it.